main/ertp/ERTP-Introduction.md

```diff
@@ -0,0 +1,117 @@
+# ERTP Introduction
+
+ERTP (*Electronic Rights Transfer Protocol*) is Agoric's token
+standard for digital assets in
+JavaScript. Using ERTP, you can easily create a wide variety of digital assets,
+all of which are transferred exactly the same way and with exactly the
+same security properties. 
+
+ERTP uses object capabilities to enforce access control. If your
+program has a reference to an object, it can call methods on that
+object. If it doesn't have a reference, it can't. 
+
+## Creating assets with ERTP
+
+In ERTP, *mints* create digital *assets*. Access to an asset type's
+mint lets you create more digital assets of that type. You can then
+store new assets in a *payment* or a *purse*. 
+- **Payments**: Assets you intend to send to someone else.
+- **Purses**: Store assets until you use them.
+
+To send assets in ERTP:
+1. Withdraw them from a purse.
+2. Put them in a payment.
+3. Send them to a recipient object as a message.
+
+To receive assets in ERTP:
+1. Create a purse for the asset type you'll receive.
+2. Get access to the asset type you'll receive. **Note:** you
+do not need access to the type's mint, just the specific asset you'll 
```

Comment:
"You do not need access to the type's mint. The Issuer (which is widely shared) will provide an empty purse for the asking."